torch/_inductor/decomposition.py

```diff
@@ -522,7 +522,9 @@ def bernoulli_p(self, p=0.5, *, generator=None):
 We put these decomps in a separate table `extra_random_decomps` to allow
 turning them on and off via `config.fallback_random`.
 """
-extra_random_decomps = get_decompositions([aten.native_dropout, aten.uniform_])
+extra_random_decomps = get_decompositions(
+    [aten.native_dropout, aten.geometric_, aten.uniform_]
```

Comment:
should we also add `aten.geometric`, after functionalization the calls to inplace op should be replaced with outplace variant

Reply:
Done, we should also do the same for other in-place random ops